velox/connectors/hive/HiveDataSink.h

```diff
@@ -209,6 +212,10 @@ class HiveInsertTableHandle : public ConnectorInsertTableHandle {
     return locationHandle_;
   }
 
+  std::optional<common::CompressionKind> compressionKind() const {
```

Comment:
Please have unit test to cover this api. Thanks!

Reply:
Do you mean creating a `HiveInsertTableHandle` and then read back the compressionKind? Where do you think we should add it?